PyPDF2/_page.py

```diff
@@ -296,7 +296,7 @@ def __init__(
 
         DictionaryObject.__init__(self)
         self.pdf: Optional[PdfReader] = pdf
-        self.indirect_ref = indirect_ref
+        self.indirect_ref = indirect_ref  # type:ignore
```

Comment:
```suggestion
        self.indirect_ref = indirect_ref  # type: ignore[assignment]
```

Reply:
Thanks, I managed to fix globally mypy yesterday. However your comments are welcomed.
Currently working on adding on "cloning" the "filtered" annotations and articles